plugins/custom-mui-theme/wrapRootElement.js

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+import { ThemeProvider } from "@material-ui/core/styles"
+
+import theme from "./theme"
+
+export const wrapRootElement = ({ element }) => {
+  console.info(`theme`, theme)
+  return <ThemeProvider theme={theme}>{element}</ThemeProvider>
```

Comment:
action: we need to have also the theme provider from styled-component. check frontend repo